jupyter-js-widgets/examples/web/index.js

```diff
@@ -2,9 +2,7 @@
 // Create a widget manager instance.
 var WidgetManager = require('./manager').WidgetManager;
 var uuid = require("jupyter-js-widgets").uuid;
-require('jquery-ui/themes/base/all.css');
-require('jupyter-js-widgets/css/widgets.min.css');
-
+require('jupyter-js-widgets/lib/widgets.css');
```

Comment:
That's another thing - I think right now a lot of stuff is distributed with the npm package, and we can trim that down to the compiled files. I think we should do that in a separate PR.